Dockerfile

```diff
@@ -12,15 +12,17 @@ ARG DEBCONF_NONINTERACTIVE_SEEN=true
 
 # Change workdir
 WORKDIR /app/
+RUN useradd -u 65532 -ms /bin/bash service && chown -R service /app
+USER 65532:65532
```

Comment:
Thank you @tonobo!

The uid of `65532` is a distroless convention but not a Docker "feature". Since we're not using distroless here, it would maybe make sense to use `nonroot` and a uid of `1001`.

Would you be open to changing that?